### ISSUE TITLE
State grpc as a dependency

### ISSUE THREAD

**Issue by Kidist-Abraham** [opencog/TinyCog#17]
We should state grpc as a dependency on the README since someone shouldn't realize that she need to install grpc after she got a building error.

**Comment by tesYolan** [comment]
@Kidist-Abraham didn't realize the tone, sorry for the edit! I undo'ed it. Besure to send a pull request! ;) Sorry.

**Comment by Dagiopia** [comment]
Yeah, grpc is a great dependency. Would also be great if we could package it as a Debian package for other users since grpc takes a whole lot of work to compile on armhf (as far as I know it doesn't even have a binary installer for the cpp api let alone for armhf)

**Comment by AmeBel** [comment]
@Dagiopia https://packages.ubuntu.com/bionic/libgrpc-dev

**Comment by Dagiopia** [comment]
@AmeBel thanks! 
@Kidist-Abraham please see if it's works fine on jessie and stretch (this is from ubuntu bionic so there may be some difference in the bins it's linked to like libc version or something)

**Comment by AmeBel** [comment]
Oh you are using debian. Here are the results for it https://packages.debian.org/search?keywords=libgrpc-dev&searchon=names&suite=all&section=all

**Comment by Kidist-Abraham** [comment]
@tesYolan I understand.
@Dagiopia and @AmeBel I will check that out.
